package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "prettierlichess",
+  "version": "1.0.0",
+  "description": "Here is the source code for both the chrome and firefox extension! It's simple but it's mine. Feel free to download, fork, use, and mess with it all you want. No attribution or anything required.",
+  "scripts": {
+    "clean-dist": "rm -rf dist/*",
```

Comment:
Hey can we replace this with something a bit more os flexible? or we are both windows so literally just "rmdir /s /q dist" would be fine with us

Reply:
I've added a dedicated webpack plugin for this (commit e705edc ),  so that it should be flexible and allow developers to use it no matter if they use windows, or linux, or mac os. Works for me on Mac OS and should work on windows too. Feel free to give it a try and tell me how it goes. 

If it does not work, I'll simply use the `rmdir` command you just suggested.

Reply:
hey looks good with that update just validated it locally. Going to merge this in and we'll try to get this rolled out to firefox again. thanks so much again!